portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,12 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="age()">
```

Comment:
Try moving this onload to the <p> component that has it. That way it will get called when that item loads and not when the page loads without it. In case in the future you plan to make the age not be on the page when you load. Or in case you go back and forth between showing it.

Reply:
for some reason it only works when it's in the body tag. i tried putting it the surrounding p, section, and 2 div tags but it didn't work

Reply:
I see. According to w3 it won't work for <p> or <div> https://www.w3schools.com/jsref/event_onload.asp